### PR TITLE
Update retrofit

### PR DESCRIPTION
**Pull request by AlexanderPrendota**
https://github.com/JetBrains/plugin-repository-rest-client/issues/34

@satamas please have a look

### CODE REVIEW

File: rest/src/main/kotlin/org/jetbrains/intellij/pluginRepository/model/xml/XmlPluginRepositoryBean.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package org.jetbrains.intellij.pluginRepository.model.xml
+
+import javax.xml.bind.annotation.XmlAccessType
+import javax.xml.bind.annotation.XmlAccessorType
+import javax.xml.bind.annotation.XmlElement
+import javax.xml.bind.annotation.XmlRootElement
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+data class XmlPluginRepositoryBean(
```
Comment (satamas) [inline rest/src/main/kotlin/org/jetbrains/intellij/pluginRepository/model/xml/XmlPluginRepositoryBean.kt]: Move xml beans into one file

Comment (AlexanderPrendota) [reply rest/src/main/kotlin/org/jetbrains/intellij/pluginRepository/model/xml/XmlPluginRepositoryBean.kt]: well, okay

File: rest/src/main/kotlin/org/jetbrains/intellij/pluginRepository/utils/RequestHelper.kt [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  catch (e: Exception) {
+    LOG.error(e.message, e)
+    restException(e.message)
+  }
+}
+
+private fun <T, R> executeAndCall(callable: Call<T>, block: (response: Response<T>) -> R): R? {
+  return try {
+    val executed = callable.execute()
+    if (executed.isSuccessful) block(executed)
```
[the remark this answers is not in the document]
Comment (AlexanderPrendota) [reply rest/src/main/kotlin/org/jetbrains/intellij/pluginRepository/utils/RequestHelper.kt]: I did as in the previous library version.  should be changed in another branch

[the remark this answers is not in the document]
Comment (AlexanderPrendota) [reply rest/src/main/kotlin/org/jetbrains/intellij/pluginRepository/utils/RequestHelper.kt]: I did as in the previous library version. should be changed in another branch

**Pull request merged**